pkg/sdkserver/sdkserver_test.go

```diff
@@ -333,7 +333,9 @@ func TestSidecarUpdateState(t *testing.T) {
 		t.Run(k, func(t *testing.T) {
 			m := agtesting.NewMocks()
 			sc, err := defaultSidecar(m)
-			assert.Nil(t, err)
+			if err != nil {
```

Comment:
I think this should be wrapped in a helper function.

Reply:
It's just a matter of taste. I prefer the current style because if we introduce a function we need to copy it in every test file and when you read code you need to dig into that additional function to check what happens there. I prefer to keep things as simple as possible. I believe that function will become more and more complex in time with additional logic, logging parameters, and so on. I would like to avoid that.
Other opinions?

Reply:
As another option, you can use the package `require` from [testify](github.com/stretchr/testify) library.
Package `require` implements the same assertions as the `assert` package but stops test execution when a test fails.
I found function [NoError](https://pkg.go.dev/github.com/stretchr/testify@v1.6.1/require?tab=doc#NoError) very useful for such cases when we want all preconditions to be completed fully or fail the test.

Reply:
@oleg-balunenko good suggestion. Actually we can either continue following `if err != nil { FailNow() }` pattern or use another package to avoid these checks. I find assert methods descriptions a bit confusing, honestly.

Reply:
No issues with pulling in `require` on my end - a little less noise in the code when wanting to fail fast.

Reply:
Yes, let us use `require`, I also have found it, while investigating alternatives, and by the way it would make code more compact.